blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -247,8 +153,15 @@ const ExtraLargePromoItem = ({ customFields }) => {
             || customFields.showByline || customFields.showDate)
           && (
             <div className="col-sm-xl-12 flex-col" style={{ position: isAdmin ? 'relative' : null }}>
-              {overlineTmpl()}
-              {headlineTmpl()}
+              <OverlineContainer
```

Comment:
todo: make sure that these components aren't in charge of whether to display. instead, they should be in charge of rendering if they're shown.